app/assets/stylesheets/modules/map.scss

```diff
@@ -0,0 +1,31 @@
+@import "variables";
+@import "mixins";
+
+.map {
+  max-width: 100%;
+  background: #f5f5f5;
```

Comment:
Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.